### PR TITLE
Ch setup travis

### PR DESCRIPTION
**Pull request by IRANKUND**
#### What does this PR do?
Setup TravisCi and Coveralls
#### Description of Task to be completed?
create .travis.yml
create coveralls.yml
#### How should this be manually tested?
add github repository in your travis and coveralls acount
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://trello.com/b/HCBFhilr/phantom-boards
#### Screenshots (if appropriate)
N/A
#### Questions:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (malaba6) [inline src/index.js]: Do not disable eslint rules. Try instead to solve the error

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 import React from 'react';
+// eslint-disable-next-line no-unused-vars
```
Comment (malaba6) [inline src/index.js]: Same thing here

File: src/index.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 import ReactDOM from 'react-dom';
-import Nav from './components/Nav';
+// import Nav from './components/Nav';
```
Comment (malaba6) [inline src/index.js]: Remove all commented out lines

File: src/index.js [hunk 4]
```diff
@@ -1,9 +1,11 @@
...
 
 
-ReactDOM.render(<Nav />, document.getElementById('app'));
+// ReactDOM.render(<Nav />, document.getElementById('app'));
```
Comment (malaba6) [inline src/index.js]: Same here

File: src/__test__/index.test.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
-import jest from 'jest';
 import sampleFunction from '../index';
 
+
 test('adds 1 + 2 to equal 3', () => {
+  
   expect(sampleFunction(1, 2)).toBe(3);
 });
```
Comment (malaba6) [inline src/__test__/index.test.js]: Lookout for eslint errors like this one. Do not ignore them plz!

**Review by malaba6 — CHANGES_REQUESTED** [review]

File: .eslintrc.json [hunk 6]
```diff
@@ -43,5 +43,18 @@
...
+			"**/*.test.jsx"
+		  ],
+		  "env": {
+			"jest": true
+		  }
+		}
+	  ]
+	
+
 }
```
Comment (maximeish) [inline .eslintrc.json]: You may add a newline at the end of each file to remove some of the es lint errors

Comment (IRANKUND) [reply .eslintrc.json]: oky am going to work on it

**Review by joshwambere — COMMENTED** [review]
thanks for the tweak in index.js it make it more readable ,

**Pull request merged**